click-to-load/click_to_load_config_reference.json

```diff
@@ -0,0 +1,50 @@
+{
+    "Tracker INC": {
+        "domains": [
+            "tracker.test",
+            "tracker.net"
+        ],
+        "excludedSubdomains": [
+            "excluded.tracker.test"
+        ],
+        "excludedDomains": [
+            {
+                "domain": "local-exception.test",
+                "reason": "Local exception"
+            }
+        ],
+        "surrogates": [
+            {
+                "rule": "tracker\\.test\\/[a-z_A-Z]+\\/sdk.js",
+                "surrogate": "surrogate.js",
+                "xray": "surrogate-xray.js"
+            }
+        ],
+        "informationalModal": {
```

Comment:
A while back I fixed a bug[1] whereby if this section was ommitted the code threw an exception instead of just considering the feature disabled. I wonder if we should test situations like that? For example, could we have an entry which doesn't include an `excludedDomains` or `excludedSubdomains` setting and test that things still work OK?

1 - https://github.com/duckduckgo/duckduckgo-privacy-extension/commit/9e48b3c3e4d562c2a4082ff38264b512b3183cec

Reply:
We sure can do it! We are doing something similar for remote config: https://github.com/duckduckgo/privacy-reference-tests/blob/main/privacy-configuration/config3_reference.json . The question is - is it an expected situation? Isn't it better to assume it's an error state that those fields are missing and disable the feature? Why only those two fields and not also `informationalModal`? WDYT?

Reply:
> Why only those two fields...

They were just examples, I agree it would be nice to have tests that verify omitting other optional fields like `informationModal` doesn't break anything.

The problem is, if these tests are only for the request blocking I'm not sure if they will catch those breakages. Request blocking works in the background "page", but lots of the CTP configuration logic happens in the content script. Could we include checking that no exceptions were thrown by the content script in your reference tests?

Reply:
I'm going to just resolve this one. After implementing other reference tests, I see that it would be down to the platform to check for things like there not being exceptions.